commands/databases_test.go

```diff
@@ -858,3 +874,54 @@ func TestDatabasesReplicaDelete(t *testing.T) {
 		assert.EqualError(t, err, errTest.Error())
 	})
 }
+
+func TestDatabaseGetSQLModes(t *testing.T) {
+	// Successful
```

Comment:
Can we wrap these in `t.Run()` if we want to add these comments around what the test case is doing?

Reply:
sure